docs/tutorials/first_app/blinky_cli.md

```diff
@@ -1,128 +1,165 @@
+[mbed] <path to ARM bin\> now set as global ARM_PATH
+
+$ mbed config --list
+[mbed] Global config:
+ARM_PATH=<path to ARM bin\>
+
 ```
 
-If you don't know the name of your target board, there are several ways to tell.
+**Note:** The same config can also be applied to IAR and GNU toolchains using `IAR_PATH` or `GCC_ARM_PATH`
 
-- Invoke `mbed detect`, and the Mbed CLI tool displays an output similar to below, where 'K64F' is the name of your target platform and COM3 is the name of the serial port that the platform connects to:
+### Code
 
+#### 1. Get the Code
+
+From your command line, import the example:
+
+```console
+$ mbed import https://github.com/ARMmbed/mbed-os-example-blinky
+$ cd mbed-os-example-blinky
 ```
-[mbed] Detected "K64F" connected to "D:" and using com port "COM3"
-```
+  
+#### 2. Compile and Program Board
 
-- Check the board information page on the list of <a href="https://developer.mbed.org/platforms/" target="_blank">Mbed Enabled boards</a>. The right side of each information page lists the name of the target.
+Invoke `mbed compile` and specify the name of your platform and your installed toolchain (`GCC_ARM`, `ARM`, `IAR`). For example, for the K64F platform and ARM Compiler 5 toolchain:
+  
+```console
+$ mbed compile --target K64F --toolchain ARM --flash
+```  
 
-- If you only have one Mbed Enabled board connected, Mbed CLI can automatically detect the target by specifing `-m detect`.
+The `--flash` argument will automatically flash the compiled program onto your board if it is connected to your computer. You can see which boards are connected with `mbed detect`. 
 
-Your PC may take a few minutes to compile your code. At the end you should get the following result:
+After the program has been flashed to the board, press the board's "reset" button and you should now see the LED blinking.
 
-```
-[snip]
-+----------------------------+-------+-------+------+
-| Module                     | .text | .data | .bss |
-+----------------------------+-------+-------+------+
-| Misc                       | 13939 |    24 | 1372 |
-| core/hal                   | 16993 |    96 |  296 |
-| core/rtos                  |  7384 |    92 | 4204 |
-| features/FEATURE_IPV4      |    80 |     0 |  176 |
-| frameworks/greentea-client |  1830 |    60 |   44 |
-| frameworks/utest           |  2392 |   512 |  292 |
-| Subtotals                  | 42618 |   784 | 6384 |
-+----------------------------+-------+-------+------+
-Allocated Heap: unknown
-Allocated Stack: unknown
-Total Static RAM memory (data + bss): 7168 bytes
-Total RAM memory (data + bss + heap + stack): 7168 bytes
-Total Flash memory (text + data + misc): 43402 bytes
-Image: .\.build\K64F\ARM\mbed-os-example-blinky.bin             
-```
+**Note** : you can get the name of the board plugged into your computer by running `mbed detect` and you can get a full list of supported toolchains / targets by running the `mbed compile --supported`
+
+
+### Debug
```

Comment:
Query: Should debugging be part of the quickstart? It feels out of scope to me.

Reply:
This is a light touch, there is a more in depth tutorial for debugging. I believe it is fine.